Formula/g/gnmic.rb

```diff
@@ -0,0 +1,28 @@
+class Gnmic < Formula
+  desc "GNMI CLI client and collector"
+  homepage "https://gnmic.openconfig.net"
+  url "https://github.com/openconfig/gnmic.git",
+      tag:      "v0.36.2",
+      revision: "a7844a6d7d47c302df1fabb845dd22e4e4b70e22"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/openconfig/gnmic/pkg/app.version=#{version}
+      -X github.com/openconfig/gnmic/pkg/app.commit=#{Utils.git_head}
+      -X github.com/openconfig/gnmic/pkg/app.date=#{time.iso8601}
+      -X github.com/openconfig/gnmic/pkg/app.gitURL=https://github.com/openconfig/gnmic
+    ]
+    system "go", "build", *std_go_args(ldflags:)
+  end
+
+  test do
+    connection_output = shell_output(bin/"gnmic -u gnmi -p dummy --skip-verify --timeout 1s -a 127.0.0.1:0 " \
+                                         "capabilities 2>&1 || true")
```

Comment:
```suggestion
    connection_output = shell_output(bin/"gnmic -u gnmi -p dummy --skip-verify --timeout 1s -a 127.0.0.1:0 " \
                                         "capabilities 2>&1", 255)
```

Reply:
Changed it to exit code 1, works now. Thanks!